tripy/tests/integration/test_dequantize.py

```diff
@@ -29,28 +29,28 @@ class TestDequantize:
     @pytest.mark.parametrize(
         "dtype", [tp.float32, tp.float16, pytest.param(tp.bfloat16, marks=skip_if_older_than_sm80)]
     )
-    def test_dequantize_int8_per_tensor(self, dtype):
+    def test_dequantize_int8_per_tensor(self, dtype, compile_fixture):
         data = [4, 8]
         input_tp = tp.Tensor(data, dtype=tp.int8)
         scale = torch.tensor(0.5, dtype=TORCH_DTYPES[dtype])
         scale_tp = tp.Tensor(scale, dtype=dtype)
-        dequantized = tp.dequantize(input_tp, scale_tp, dtype)
+        dequantized = compile_fixture(tp.dequantize, input_tp, scale_tp, dtype, exclude_arg_names=["scale"])
```

Comment:
I think we can drop `exclude_arg_names` by using the same approach you used in `cast`:
```py
def func(input):
    return tp.dequantize(input, scale_tp, dtype)

dequantized = compile_fixture(func, input_tp)
```
That will simplify the fixture implementation considerably IMO.